### ISSUE TITLE
CodeCov doesn't provide test coverage reports for some pull requests

### ISSUE THREAD

**Issue by proshin-roman** [yegor256/cactoos#846]
**What's wrong**
CodeCov doesn't provide test coverage reports for some pull requests: https://github.com/yegor256/cactoos/pull/842 and https://github.com/yegor256/cactoos/pull/835 pointing to [this article](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).

**What's expected**
CodeCov should provide a test coverage report for each pull request.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by llorllale** [comment]
@proshin-roman fixed in fc157ce6941a0881faa9d11256bea485654172bc, thanks

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@38e2c9bb-8f57-4fb1-8558-17fff4f33af4, thread: AsyncFlush-7
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/close_job.groovy
sun.reflect.NativeMethodAccessorImpl[-2] java.lang.reflect.InvocationTargetException: null
com.zerocracy.farm.strict.StrictProject[111] java.lang.IllegalArgumentException: File "blanks/proshin-roman.xml" is not accessible in "PMO"
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): CID: 1291915207, Type: 'Close job', Author: llorllale
